core/src/main/java/com/netflix/conductor/core/config/TracingProperties.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2023 Netflix, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.netflix.conductor.core.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("conductor.tracing")
+public class TracingProperties {
+
+    private Boolean enabled = false;
+
+    private String tracesEndpoint = null;
+
+    private String metricsEndpoint = null;
+
+    private String sentryDsn = null;
+
+    private Double tracesSamplingRate = 1.0;
+
+    public Boolean getEnabled() {
+        return this.enabled;
+    }
+
+    public void setEnabled(Boolean enabled) {
```

Comment:
didn't see where we define whether it is or isn't enabled, may you point me to it?

Reply:
Curious due to local environment execution

Reply:
All of the properties in this file can be defined in the config-local.properties, eg:
```
conductor.tracing.enabled=true
conductor.tracing.tracesEndpoint=http://localhost:4318/v1/traces
conductor.tracing.metricsEndpoint=http://localhost:4318/v1/metrics
conductor.tracing.sentryDsn=...
```